### ISSUE TITLE
Wrong dialog position when browser window isn't fullscreen

### ISSUE THREAD

**Issue by mirkoperillo** [mirkoperillo/resting#104]
A user signals that the "Help Resting" dialog is presented all on the right of screen instead of in the middle.

This happens on MacOS ( Catalina in particular).
I don't know if this problem exists only for this dialog of it happens to all application dialogs.

I'm not able to reproduce the bug, because I'm not a Mac user.

🙏 Looking for some mac user to confirm the problem and if possible to have a PR to fix it 🙏

**Comment by alexmario74** [comment]
I just try now on Windows and Firefox 71.0 (64 bit) and I saw the dialog on the right now.

So I think is just about Firefox update and not the operating system.

**Comment by alexmario74** [comment]
I tested on my Mac Catalina v10.15.1 Firefox v71.

I confirm that the dialog is appearing on the right, but only if you have the window not in fullscreen.

When you have the window in fullscreen the dialog is in the center.

I think is not an issue with mac os only.

Here a fullscreen:

![Schermata 2019-12-18 alle 18 49 24](https://user-images.githubusercontent.com/1519470/71110207-21f2f300-21c7-11ea-9828-919eba430bf1.png)

Here with the issue:

![Schermata 2019-12-18 alle 18 50 05](https://user-images.githubusercontent.com/1519470/71110277-418a1b80-21c7-11ea-9e60-812cd0112a3d.png)

**Comment by alexmario74** [comment]
Here I have a screenshot from windows that show the issue is there too:

![image](https://user-images.githubusercontent.com/1519470/71110645-00ded200-21c8-11ea-94ec-280d01cf7421.png)

**Comment by alexmario74** [comment]
I think I found the issue in several part of the code:

_app.js_:
```javascript
    const screenWidth = screen.width;
    const dialogLeftPosition = screenWidth / 2  - 200;
    $('div.dialog').css('left', dialogLeftPosition+'px');
```
so in _addFolderVm.js_:
```javascript
    $(() => {
      const screenWidth = screen.width;
      const dialogLeftPosition = screenWidth / 2  - 200;
      $('.dialog').css('left', dialogLeftPosition+'px');
    });
```

and in _bookmarksVm.js_:
```javascript
    $(() => {
      const screenWidth = screen.width;
      const dialogLeftPosition = screenWidth / 2  - 200;
      $('div.dialog').css('left', dialogLeftPosition+'px');
```
All that code is running on when the document is ready and the selector `$('div.dialog')` or `$('.dialog')` are matching all the dialogs included the feedback.

Another issue is the use of `screen.width` instead of `window.innerWidth`, as the first is always the size of the screen and not the size of the window. The 2 values are the same only when windows is fullscreen.

**Comment by mirkoperillo** [comment]
@alexmario74  thank you for the detailed feedback and analysis.

I have to change the title of this issue

**Comment by alexmario74** [comment]
If you don't mind I would work on this.

**Comment by mirkoperillo** [comment]
@alexmario74 it would be amazing

**Comment by mirkoperillo** [comment]
fixed by d262fd24151e40e81c8385b41a31030d3aa33bc0
